bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -223,6 +325,12 @@ private void processSettings(String content) {
             ipCameraHandler.setChannelState(CHANNEL_ENABLE_MOTION_ALARM, OnOffType.OFF);
         }
 
+        // Handle MotionDetectLevel alarm
+        if (content.contains("table.MotionDetect[0].Level=")) {
+            String value = ipCameraHandler.returnValueFromString(content, "table.MotionDetect[0].Level=");
+            ipCameraHandler.setChannelState(CHANNEL_MOTION_DETECTION_LEVEL, PercentType.valueOf(value));
```

Comment:
Look strange to use a PercentType while tjos channel type expects a number amongst 0, 1, 2, 3, 4, 5 or 6.
What do I miss ?

Reply:
I would expect a DecimalType with value 0, 1, .... or 6.

Reply:
Agree and changed.